app/views/claims/support/claims/index.html.erb

```diff
@@ -4,7 +4,7 @@
 <div class="govuk-width-container">
   <h1 class="govuk-heading-l"><%= t(".heading", count: @pagy.count) %></h1>
 
-  <%= govuk_button_to(t(".download_csv"), download_csv_claims_support_claims_path, method: :get) %>
+  <%= govuk_link_to t(".download_csv"), download_csv_claims_support_claims_path(**request.query_parameters), class: "govuk-button", method: :get %>
```

Comment:
Do we have a spec for this behaviour. Having a list of claims adding some filters which changes the list of filters and then exporting those filtered claims to CSV?

Do you think we need something like this?